lib/node_modules/@stdlib/math/base/complex/divide/benchmark/benchmark_smith.js

```diff
@@ -0,0 +1,39 @@
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var isArray = require( '@stdlib/assert/is-array' );
+var cdivSmith = require( './cdiv_smith.js' );
+
+
+// MAIN //
+
+bench( "complex division using Smith's algorithm", function benchmark( b ) {
```

Comment:
We have a benchmark description convention which allows us to automatically generate cross-comparison plots among benchmarks. This convention is orthogonal to our `tape` convention. Here, the description would be 

``` javascript
bench( pkg+'::smiths_algorithm', ... );
```
where `pkg` is the package name, allowing us to group benchmarks by package, `::` indicates a sub-namespace, and `smiths_algorithm` is the namespace.

This is not documented anywhere and is one of the conventions which needs to be included in the benchmarks guide. Sorry about that.